docs/ota-client-guide/modules/ROOT/pages/push-images-with-bitbake.adoc

```diff
@@ -0,0 +1,17 @@
+== Push new images with BitBake
+
+After you rotate your software signing keys, you can upload new software images using BitBake or xref:upload-large-binary.adoc[garage-sign].
+
+*To upload software images using BitBake:*
```

Comment:
This feels a little bit imprecise/possibly misleading. You don't really have a choice here: if you're building Yocto images they get uploaded as part of the build, and if you want to upload binary packages you have to use garage-sign.

For the Yocto case, the only thing that changes is that instead of using credentials downloaded from the portal, you have to use a credentials file you create yourself using garage-sign.

Reply:
Modified the title and the intro sentence.